evm/generators.go

```diff
@@ -1175,13 +1455,40 @@ func {{.DeployHandler.HandlerName}}() *cobra.Command {
 	cmd.Flags().Uint64Var(&gasLimit, "gas-limit", 0, "Gas limit for the transaction")
 	cmd.Flags().BoolVar(&simulate, "simulate", false, "Simulate the transaction without sending it")
 	cmd.Flags().UintVar(&timeout, "timeout", 60, "Timeout (in seconds) for interactions with the JSONRPC API")
+	cmd.Flags().BoolVar(&useSafe, "safe", false, "Create a Safe proposal instead of directly executing the transaction")
```

Comment:
I think we don't need to have both `--safe` and `--safe-address`.

Let us just set `--safe` to the address. If the value is not the empty string, then we will enter the safe flow, otherwise we do an EOA transaction.